php/Dockerfile.j2

```diff
@@ -43,8 +43,8 @@ RUN echo "Starting ..." && \
     pecl install memcached-${MEMCACHED_VERSION} && \
     docker-php-ext-enable memcached && \
     docker-php-ext-configure pgsql -with-pgsql=/usr/local/pgsql && \
-    pecl install ssh2-${SSH2_VERSION} && \
-    docker-php-ext-enable ssh2 && \
+    # pecl install ssh2-${SSH2_VERSION} && \
```

Comment:
please add before `# waiting for https://github.com/php/pecl-networking-ssh2/pull/36 to be merged`